autoprotocol/container_type.py

```diff
@@ -1071,6 +1071,42 @@ def row_count(self):
     cat_no="3657",
 )
 
+#:
+ROUND384LV = ContainerType(
+    name="384-well round-bottom low-volume non-binding-surface Corning-4513 plate",
+    well_count=384,
+    well_depth_mm=6.58,
+    well_volume_ul=Unit(20.0, "microliter"),
+    well_coating="Corning NBS(nonbinding surface)",
```

Comment:
im pretty sure this can be anything - i just went to the vendor site and transcribed from there